### PR TITLE
feat(perf-docs): reorganize and regenerate screenshots

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
https://docs.sentry.io/performance/getting-started/?platform=python

**In Progress:**
Screen Shot 2020-06-25 at 10 02 51 AM

### CODE REVIEW

**Comment by doralchan** [comment]
@dcramer I should be done with this today

File: src/collections/_documentation/performance-monitoring/discover-queries/query-builder.md [hunk 1]
```diff
@@ -305,4 +305,5 @@ Deleting a saved query is irreversible.
 
 On the Discover Homepage, each saved query card has an ellipsis that will open a context menu. Delete the query from here. You can also delete the query within Query Results view by clicking the trash can in the upper right. 
 
-For more details on how to view saved queries, see [documentation on Discover](/performance/discover/)
+query-builder.md
```
Comment (dashed) [inline src/collections/_documentation/performance-monitoring/discover-queries/query-builder.md:308]: looks like a stray change that shouldn't be here.

**Comment by MimiDumpling** [comment]
🚀 Pushed some changes. Overall, it's good to go

**Review by dashed — APPROVED** [review]
👍 🌮

**Pull request merged**